financial-connections-example/src/main/java/com/stripe/android/financialconnections/example/settings/FlowSetting.kt

```diff
@@ -16,14 +16,32 @@ data class FlowSetting(
 
     override fun paymentIntentRequest(body: PaymentIntentBody): PaymentIntentBody = body
 
-    override fun valueUpdated(currentSettings: List<Setting<*>>, value: Flow): List<Setting<*>> =
-        replace(currentSettings, this.copy(selectedOption = value))
-
     override fun convertToString(value: Flow): String {
         return value.apiValue
     }
 
     override fun convertToValue(value: String): Flow {
         return Flow.fromApiValue(value)
     }
+
+    override fun valueUpdated(currentSettings: List<Setting<*>>, value: Flow): List<Setting<*>> {
```

Comment:
Can you elaborate why this change is necessary? I don’t fully understand what’s going on here.

Reply:
The code is not ideal, but basically removes the toggle for non payment flows (uses the same approach that we use to show public / private key fields when merchant is `Other`).